### PR TITLE
Feature/GenericInputs 

### PR DESCRIPTION
**Pull request by geekysilento**
image

Ignore the styling inconsistencies, will address them in a future commit

To do:

- Time input field
- Fetch countries for telephone, maybe using a external library like https://www.npmjs.com/package/react-phone-number-input would be better? 

Any advice is appreciated.

### CODE REVIEW

File: components/inputs/Visualizer.tsx
Comment (GetPsyched) [inline components/inputs/Visualizer.tsx:1]: is this gonna be dumped later?

File: components/inputs/date.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import * as React from 'react';
```
Comment (GetPsyched) [inline components/inputs/date.tsx:3]: IDT this is needed

File: components/inputs/select.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { toast } from '@/components/ui/use-toast';
+
+const items = [
+  {
+    id: 'recents',
+    label: 'Recents',
+  },
```
Comment (GetPsyched) [inline components/inputs/select.tsx:24]:
```suggestion
  { id: 'recents', label: 'Recents' },
```

File: components/inputs/telephone.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+'use client';
+import { useState } from 'react';
```
Comment (GetPsyched) [inline components/inputs/telephone.tsx:2]:
```suggestion

import { useState } from 'react';
```

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: components/inputs/Visualizer.tsx
Comment (geekysilento) [reply components/inputs/Visualizer.tsx:1]: yes, that component is just to visualize the inputs

File: app/page.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import Visualizer from '@/components/inputs/Visualizer';
 export default function Home() {
   return (
     <div>
-      <Visualizer />
```
Comment (GetPsyched) [inline app/page.tsx]: uh?

**Pull request closed without merging**